resources/views/home.blade.php

```diff
@@ -59,7 +59,7 @@
                             <ul class="post-footer">
                                 <li>
                                     <a href="{{ route('login') }}"><i class="ion-heart"></i>
-                                        {{ $post->favorite_to_users->count() }}
+                                        {{ $post->usersFavorite()->count() }}
```

Comment:
```suggestion
                                        {{ $post->usersFavorite->count() }}
```
dùng usersFavorite nhé ko nó lại tạo thêm truy vấn.